packages/vue-spec-renderer/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "@kong/portal-vue-spec-renderer",
+  "version": "0.0.1",
+  "description": "",
+  "type": "module",
+  "main": "./dist/vue-spec-renderer.umd.js",
+  "module": "./dist/vue-spec-renderer.es.js",
+  "types": "./dist/types/index.d.ts",
+  "scripts": {
+    "dev": "cross-env USE_SANDBOX=true vite",
+    "build": "run-s typecheck build:package build:types",
+    "build:package": "vite build",
+    "build:types": "vue-tsc -p './tsconfig.build.json' --emitDeclarationOnly",
+    "preview:package": "vite preview --port 4173",
+    "preview": "cross-env USE_SANDBOX=true PREVIEW_SANDBOX=true run-s build:package preview:package",
+    "lint": "eslint '**/*.{js,jsx,ts,tsx,vue}' --ignore-path '../../.eslintignore'",
+    "lint:fix": "eslint '**/*.{js,jsx,ts,tsx,vue}' --ignore-path '../../.eslintignore' --fix",
+    "stylelint": "stylelint --allow-empty-input './src/**/*.{css,scss,sass,less,styl,vue}'",
+    "stylelint:fix": "stylelint --allow-empty-input './src/**/*.{css,scss,sass,less,styl,vue}' --fix",
+    "typecheck": "vue-tsc -p './tsconfig.build.json' --noEmit",
+    "test:component": "BABEL_ENV=cypress cross-env FORCE_COLOR=1 cypress run --component -b chrome --spec './src/**/*.cy.ts' --project '../../.'",
+    "test:unit": "cross-env FORCE_COLOR=1 vitest run",
+    "test:unit:open": "cross-env FORCE_COLOR=1 vitest --ui"
+  },
+  "keywords": [],
+  "author": {
+    "name": "Kong Inc.",
+    "url": "https://konghq.com"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/Kong/portal-components/issues"
+  },
+  "repository": {
+    "url": "https://github.com/Kong/portal-components/tree/main/packages/vue-spec-renderer"
+  },
+  "peerDependencies": {
+    "vue": "^3.2.45"
+  },
+  "devDependencies": {
+    "@modyfi/vite-plugin-yaml": "^1.0.3"
+  },
+  "dependencies": {
+    "@kong/swagger-ui-web-component": "^1.0.2"
+  },
+  "volta": {
+    "node": "18.12.1"
+  }
```

suggestion: This should only be defined in the root
```suggestion
  "volta": {
    "extends": "../../package.json"
  }
```